src/components/molecules/AdminFooter/AdminFooter.tsx

```diff
@@ -0,0 +1,32 @@
+const navigation = [
+  { name: "About", href: "#about" },
+  { name: "Blog", href: "#blog" },
+  { name: "Jobs", href: "#jobs" },
+  { name: "Press", href: "#press" },
+  { name: "Accessibility", href: "#accessibility" },
+  { name: "Partners", href: "#partners" },
+];
+
+const renderedMenuItems = navigation.map((item) => (
+  <div key={item.name} className="px-5 py-2">
+    <a href={item.href} className="text-base text-gray-500 hover:text-gray-900">
+      {item.name}
+    </a>
+  </div>
+));
+
+export const AdminFooter = () => (
```

Comment:
Can we move this and similar components under `./src/components/admin` for easier inventory tracking? Might usually benefit from a rename, though not sure if in this case it will overload the name for attendee and admin versions